webpack/actions/search_results_actions.js

```diff
@@ -63,3 +67,34 @@ export function setLastSearch(searchTerms=null, type=null, programFilter=[], sys
     payload: { type: type, search: searchTerms, programs: programFilter, systems: systemFilter, mystuff: myStuffFilter, page: page }
   };
 }
+
+// Everything in elasticsearch has codes, with code and codeSystem. The models
+// expect their own thing ResponseSet.responses, Question.concepts with value
+// and codeSystem. This function will transform the elasticsearch results into
+// the structure expected by the rest of the react application.
+function unelasticsearchResults(results) {
+  if (results.responseSets) {
```

Comment:
It looks like the logic that this function replaces was removed from mapStateToProps in QuestionEditContainer, but not in ResponseSetEditContainer